package.json

```diff
@@ -8,9 +8,17 @@
     "lint": "vue-cli-service lint"
   },
   "dependencies": {
+    "bootstrap": "^4.6.0",
+    "bootstrap-vue": "^2.21.2",
     "core-js": "^3.6.5",
-    "vue": "^2.6.11",
-    "vue-router": "^3.2.0"
+    "css-loader": "^5.2.4",
```

Comment:
css-loader & style-loader is not required. The css loaders are automatically handled by vue-cli